### DISCUSSION TITLE
Does mlua support luau's type annotations?

### DISCUSSION THREAD

**Discussion by jqnatividad** [mlua-rs/mlua#255] [Q&A] [answered]
One of luau's more compelling features is [type annotation](https://luau-lang.org/syntax#type-annotations).

Does mlua `luau` support allow it?

**Comment by khvzak** [comment, marked as answer]
type annotations are supported (but luau compiler simply ignores it). I think simiar is in python.
To check the source code with type anotations you need to run an additional tool called `luau-analyzer`. It's shipped with luau [distribution](https://github.com/Roblox/luau/releases)

**Reply by jqnatividad** [reply]
Thanks for the quick reply! I was trying it out with type annotations and I think it actually caused a syntax error, so I removed it.

Anyway, I didn't know the type annotations were primarily meant to be used with the analyzer, so thanks for the tip!
